pages/docs/docs.md

```diff
@@ -1324,6 +1348,21 @@ or use parens like so:
     ):
       doStuff()
 
+### Decimals
+
+Numbers in LightScript must begin and end with a number character. That means they
```

Comment:
Yeah, me too. I call them "Numbers with decimals". I might reword the section as:

```
### Numbers with decimals

Numbers in LightScript cannot begin or end with a `.`: